requirements.txt

```diff
@@ -1,22 +1,28 @@
 astroid==2.2.5
 atomicwrites==1.3.0
 attrs==19.1.0
+autopep8==1.4.4
 coverage==4.5.3
+dj-database-url==0.5.0
```

Comment:
is the package used in the project?

Reply:
For now it's not needed but cause i have not added any db url configutaions

Reply:
please remove it and push the updates.